web/themes/contrib/civictheme/civictheme_library/components/style.editor.scss

```diff
@@ -3,41 +3,18 @@
 //
 // stylelint-disable scss/at-import-partial-extension
 
-@use 'sass:meta';
-@use 'sass:map';
-@use '00-base/variables';
+// Need to add variables outside of scoping class to avoid compilation errors for variables with !global flags.
```

Comment:
@awset 
Could you please explain what is going on with this change. You are suggesting to significantly change how variables are imported and i need to understand why

Reply:
I moved the `@import '00-base/**/!(*.stories|variables|_variables.*|reset).scss';` under `.ck-editor__editable` is for preventing the class in that 00-base to be applied to the BE theme.